chtf/chtf.sh

```diff
@@ -20,7 +20,9 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-CHTF_VERSION='1.2.1'
+# load version
+VERSION_FILE="$(dirname "${BASH_SOURCE[0]}")"/VERSION
```

Comment:
Oh, only now realised that this breaks zsh. So I guess we need to use `$0`. I'll fix it directly to master.

Reply:
That actually would break bash when sourcing file from outside of `chtf` folder:

```
bash-3.2$ source chtf/chtf.sh
+ source chtf/chtf.sh
+++ dirname /bin/bash
++ VERSION_FILE=/bin/VERSION
+++ cat /bin/VERSION
cat: /bin/VERSION: No such file or directory
++ CHTF_VERSION=
++ : /usr/local/Caskroom
```
I think this might work in both though:

```SCRIPT_DIR=`dirname ${BASH_SOURCE[0]-$0}```

Reply:
Sigh, I was too hurry. `$0` won't work in zsh either. Yet another try: 56966e9894d5e

Reply:
That actually looks the cleanest, worked for me with bash from different locations 👍